runtime/descriptor.go

```diff
@@ -98,24 +105,54 @@ func propertySet(f *Frame, desc, inst, value *Object) *BaseException {
 
 // makeStructFieldDescriptor creates a descriptor with a getter that returns
 // the field given by fieldName from t's basis structure.
-func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object {
+func makeStructFieldDescriptor(t *Type, fieldName, propertyName string, fieldType fieldDescriptorType) *Object {
```

Comment:
I though about testing against ``.canSet()`` as seen on http://stackoverflow.com/a/6396678/798575. Is it better to check ``field.PkgPath`` instead? Please confirm